project.clj

```diff
@@ -45,7 +45,9 @@
                  [com.google.apis/google-api-services-analytics       ; Google Analytics Java Client Library
                   "v3-rev139-1.22.0"]
                  [com.google.apis/google-api-services-bigquery        ; Google BigQuery Java Client Library
-                  "v2-rev342-1.22.0"]
+                  "v2-rev324-1.22.0"]
+                 [com.h2database/h2 "1.4.193"]                        ; embedded SQL database
```

Comment:
oops, thanks for catching that, it came from rebasing this onto the current master when starting this project